tests/test_builder.py

```diff
@@ -40,7 +39,7 @@ def get_sphinx_output(srcdir, outdir, docname):
     app.build()
     path = os.path.join(outdir, docname + '.spelling')
     try:
-        with codecs.open(path, 'r') as f:
+        with open(path) as f:
```

Comment:
I like having the mode be explicit in the open call. Could you restore that part? Otherwise, this looks good.

Reply:
Works for me :+1:

Done in the latest revision.